plugins/git/functions

```diff
@@ -41,6 +41,21 @@ git_build_app_repo() {
   fi
 }
 
+suppress_output() {
```

Comment:
Cool great. Going forward, outputting all logs to a file and the error output from a failing command to stdout seems reasonable.